src/backend/controllers/AuthController.js

```diff
@@ -15,42 +15,42 @@ const sign = require("jwt-encode");
  * */
 
 export const signupHandler = function (schema, request) {
```

Comment:
Looks like you beautified controller files, unnecessarily it added indents and now showing in changed file in pr!